### ISSUE TITLE
Decouple stackFiltering from stackShortening

### ISSUE THREAD

**Issue by kriskowal** [endojs/endo#913]
Talking with @michaelfig, it seemed useful to decouple the lockdown options for stackFiltering (which would be responsible just for removing lines inside the SES machinery from stack traces) and stackShortening (which would just be responsible for abbreviating file paths to a suffix).

cc @erights

**Comment by kriskowal** [comment]
Seemingly completed by https://github.com/endojs/endo/pull/2747. Please reopen if you disagree that `stackFiltering: 'shorten-paths'` is a distinct feature.

**Comment by erights** [comment]
Yes. Likewise `stackFiltering: 'omit-frames'` is the opposite distinct feature.
